### DISCUSSION TITLE
The page Next.js Speed Insights is not working

### DISCUSSION THREAD

**Discussion by s5s5** [vercel/community#2136] [Bug Report]
https://nextjs.org/learn/seo/monitor/nextjs-speed-insights

<img width="1727" alt="image" src="https://user-images.githubusercontent.com/473706/233296628-a7d96d56-3edf-457d-8f21-497725e194c6.png">

**Comment by swarnava** [comment]
Hi,

Thank you for reporting this. I have reported this to Next.js team to take a closer look.
